yt/fields/tests/test_angular_momentum.py

```diff
@@ -0,0 +1,33 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2016, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+import numpy as np
+
+import yt
+from yt.testing import \
+    assert_allclose_units, \
+    requires_file
+
+@requires_file('output_00080/info_00080.txt')
```

Comment:
Any chance we can run this test using e.g. `fake_amr_ds` or some other dataset that doesn't need to download. I don't think anything about this test is specific to this RAMSES output?

Reply:
I'm not sure why but the test passes even without the fix when using `fake_amr_ds`. Any guess why?